src/webpack/webpack.config.js

```diff
@@ -0,0 +1,141 @@
+const Autoprefixer = require('autoprefixer');
+const PostCSSAssetsPlugin = require('postcss-assets-webpack-plugin');
+const PostCSSCustomProperties = require('postcss-custom-properties');
+const path = require('path');
+const rtl = require('postcss-rtl');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CleanPlugin = require('clean-webpack-plugin');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const aggregateTranslations = require('../../scripts/aggregate-translations/aggregate-translations');
+const merge = require('webpack-merge');
+
+const defaultWebpackConfig = (env = {}, argv = {}) => {
+  const production = argv.p;
+  const disableAggregateTranslations = env.disableAggregateTranslations;
+  const processPath = process.cwd();
+  /* Get the root path of a mono-repo process call */
+  const rootPath = processPath.includes('packages') ? processPath.split('packages')[0] : processPath;
+
+  const resolveModules = ['node_modules'];
+  if (!disableAggregateTranslations) {
+    aggregateTranslations({ baseDirectory: rootPath });
```

Comment:
Should we open up env to allow an `aggregateOptions` key to pass down to `aggregateTranslations`?